src/main/java/org/openRealmOfStars/player/tech/TechFactory.java

```diff
@@ -96,7 +96,7 @@ private TechFactory() {
    * Rare combat tech names for level 6.
    */
   public static final String[] COMBAT_RARE_TECH_LEVEL7_NAMES = {
-      "Ion cannon Mk3", "Plasma cannon Mk4"};
+      "Ion cannon Mk3", "Plasma cannon Mk4", "Callisto multicannon Mk1"};
```

Comment:
Rare tech is special tech which is gained from space anomalies or other events. This would need to be in "regular" tech and one weapon tech for each level. "Callisto multicannon Mk1" for level 1, "Callisto multicannon Mk2" for level 2 and so on. On level 6 and 9 there is bigger upgrade on weapons. Other weapons gain weapon range, but maybe this would gain more accuracy.